cvpy/tests/test_cvat_project.py

```diff
@@ -72,6 +75,69 @@ def test_cvat_project_add_task(self):
         self.assertEqual(tasks[0], task_1)
         self.assertEqual(tasks[1], task_2)
 
+    # Post images to a project.
+    def test_cvat_project_post_images(self):
+
+        # Setup CAS and create an ImageTable.
+        cas_connection = swat.CAS(TestCVATProject.cas_host, TestCVATProject.cas_port, protocol='http')
+        cas_connection.loadactionset('image')
+        cas_connection.addcaslib(name='dlib', 
+                                      activeOnAdd=False, 
+                                      path=TestCVATProject.datapath, 
+                                      dataSource='PATH',
+                                      subdirectories=True)
+
+        cas_table_encoded = cas_connection.CASTable('cas_table_encoded')
+        cas_connection.image.loadimages(labellevels=5,
+                                        casout=cas_table_encoded,
+                                        caslib='dlib',
+                                        decode=False)
+
+        cas_table_decoded = cas_connection.CASTable('cas_table_decoded')
+        cas_connection.image.loadimages(labellevels=5,
+                                        casout=cas_table_decoded,
+                                        caslib='dlib',
+                                        decode=True)
+
+        image_table_encoded = ImageTable(cas_table_encoded) 
+        image_table_decoded = ImageTable(cas_table_decoded) 
+
+        # Create a CVATProject.
+        url = TestCVATProject.cvat_url
+
+        project_name = 'Test Project'
+        annotation_type = AnnotationType.OBJECT_DETECTION
+
+        mountain_label = AnnotationLabel(name='Mountain', color='orange')
+        person_label = AnnotationLabel(name='Person', color='green')
+
+        labels = [mountain_label, person_label]
+
+        credentials = Credentials(self.cvat_username, self.cvat_password)
+        cvat_project = CVATProject(url=url, cas_connection=cas_connection, credentials=credentials,
+                                   project_name=project_name, annotation_type=annotation_type,
+                                   labels=labels)
+
+        # Post the images to the CVATProject.
+        cvat_project.post_images(image_table_encoded)
+        cvat_project.post_images(image_table_decoded)
```

Comment:
I think we should add a REST call here to get the list of tasks from CVAT that are associated with cvat_project, and assert that those tasks have the same names, ids, etc. as those on cvat_project.get_tasks().

Reply:
Should I also pull the images from CVAT and compare them to the images I uploaded? I CAN do that and was originally planning on doing all of that verification, but then I had the thought that if we posted the images without getting an exception (meaning all response.status_codes were 'good') that everything worked as intended from the CVAT point of view and any pulling of info from CVAT was really testing CVAT and not our code.

Reply:
Makes sense. I've just uploaded the updates (including the additions to the unit test). Let me know what you think!